packages/__docs__/src/index.js

```diff
@@ -23,16 +23,15 @@
  */
 
 import React from 'react'
-import ReactDOM from 'react-dom'
+import { createRoot } from 'react-dom/client'
 
 import { App } from './App'
 import { canvas } from '@instructure/ui-themes'
 import { InstUISettingsProvider } from '@instructure/emotion'
 import '../globals'
 
-ReactDOM.render(
```

Comment:
I can still see warnings, because the Alert component still has `ReactDOM.render` in it.

Reply:
yep, this just fixes them in the docs. One in `Preview` was really annoying, it was triggering the warning on every code change in the code editor.
This is more tricky to fix because:   
- we need to import `createRoot` asynchronously like here: https://github.com/instructure/instructure-ui/blob/master/packages/ui-test-sandbox/src/utils/reactComponentWrapper.tsx#L27 This might require other code changes too. 
- Removing a root can trigger sometimes scary sounding warnings, I have yet to figure out how to deal with them properly: https://stackoverflow.com/questions/73043828/how-to-unmount-something-created-with-createroot-properly 

So I'd do this in a separate ticket. Also AFAIK this just renders a very small subtree in the legacy slow mode, so it's not a big practical issue.